agent/exploits/cve_2014_0780.py

```diff
@@ -26,16 +27,20 @@ class CVE20140780Exploit(definitions.Exploit):
     CVE-2014_0780: Indusoft Web Studio Directory Traversal.
     """
 
+    def __init__(self) -> None:
+        self.session = requests.Session()
```

Comment:
Why use self.session. This has a risk of polluting the session across runs. We should init the session in the execution of the rule and that's it.